app/templates/components/user/display-username.hbs

```diff
@@ -0,0 +1,5 @@
+{{#if username}}
+  {{#link-to data-test-username 'slugged-route' username class="username"}}{{username}}{{/link-to}}
+{{else if githubUsername}}
+  GitHub user <a data-test-github-username href={{githubUserUrl}} class="username">{{githubUsername}}</a>
```

Comment:
Might not be a bad thing to add `(via GitHub)` here, but not sure how nice it would look with the "timestamp" that goes after in some cases.

Reply:
Right now it reads `GitHub user joshsmith commented 21 hours ago`. I think after that we could add `– from GitHub` with a link to the GitHub app. But that wouldn't be part of this component then. I also plan in the avatar to add the GitHub logo in the lower right corner, similar to how GitHub does with our app's logo, to make it even clearer.

Reply:
I have no idea how I didn't see the `Github user` part here. Just ignore my comment